### PR TITLE
feat: Disable taoGroups for GlobalManager role if testCenter is installed

### PR DESCRIPTION
**Pull request by yaraslau-kavaliou**
Ticket - https://oat-sa.atlassian.net/browse/REL-1054

### CODE REVIEW

File: migrations/Version202303301430191871_taoTestCenter.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->updateACL(false);
+    }
+
+    private function updateACL(bool $include)
+    {
+        $role = new core_kernel_classes_Resource('http://www.tao.lu/Ontologies/TAOGroup.rdf#GroupsManagerRole');
```
Comment (gabrielfs7) [inline migrations/Version202303301430191871_taoTestCenter.php]: This ACL will apply to all community, not just to ignite/pro. Shall we have use an ACL command instead for this? 

The fact that groups are not working with test-center is a bug that we should fix, but I am not sure we want to disable it to the whole community (at least I did not find it in the ticket ACs).

File: scripts/install/HideTaoGroups.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+use tao_models_classes_RoleService;
+
+class HideTaoGroups extends InstallAction
+{
+    public function __invoke($params)
+    {
+        self::updateACL(false);
+    }
+
+    public static function updateACL(bool $include)
```
Comment (augustas) [inline scripts/install/HideTaoGroups.php]: I'd suggest 2 methods instead 1 with a boolean argument

File: scripts/install/HideTaoGroups.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    {
+        $role = new core_kernel_classes_Resource('http://www.tao.lu/Ontologies/TAOGroup.rdf#GroupsManagerRole');
```
Comment (augustas) [inline scripts/install/HideTaoGroups.php]: Lets move the string to a `const`

File: scripts/install/HideTaoGroups.php (hunk 2 above)
Comment (yaraslau-kavaliou) [reply scripts/install/HideTaoGroups.php]: done

File: scripts/install/HideTaoGroups.php (hunk 3 above)
Comment (yaraslau-kavaliou) [reply scripts/install/HideTaoGroups.php]: done

File: scripts/install/HideTaoGroups.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+class HideTaoGroups extends InstallAction
+{
+    private const GROUP_MANAGER_ROLE = 'http://www.tao.lu/Ontologies/TAOGroup.rdf#GroupsManagerRole';
+
+    public function __invoke($params)
+    {
+        self::hideGroupsFromGlobalManager();
+    }
+
+    public static function hideGroupsFromGlobalManager()
```
Comment (augustas) [inline scripts/install/HideTaoGroups.php:39]: Not sure if `static` is necessary here?

**Review by augustas — APPROVED** [review]
I approve the PR, yet as mentioned in another comment, `static` invocation doesn't seem to be necessary in `HideTaoGroups` :)

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**